package.json

```diff
@@ -44,7 +44,7 @@
 		"morgan": "1.7",
 		"multer": "1.2",
 		"multipipe": "1.0",
-		"nodemailer": "2.6",
```

Comment:
Do you mean just specifying 3.0 instead of 3.0.2?  That is fine.  There was nothing particular in 3.0 over 2.6 if that is what you are asking, but I didnt see any issues with 3.0 so made sense to upgrade while I was retesting all the email stuff.